saleor/discount/utils/order.py

```diff
@@ -132,9 +136,16 @@ def _copy_unit_discount_data_to_order_line(
             line = line_info.line
             discount_amount = sum([discount.amount_value for discount in discounts])
             unit_discount_amount = discount_amount / line.quantity
+
+            # Manual discounts can be added without a reason provided. To not lose info about
```

Comment:
Maybe extract that to the function. I can see that this comment and code is used multiple times, WDYT?

Reply:
Moved to separate function